Library/.rubocop.yml

```diff
@@ -69,10 +74,6 @@ Style/PercentLiteralDelimiters:
         '%W': '[]'
         '%x': '()'
 
-# comma on last element is editor friendly
-Style/TrailingComma:
-    EnforcedStyleForMultiline: comma
```

Comment:
I prefer this as well. However it will append comma for method calling, e.g. `foo(a, b, )`, which is syntax error.